test/common/deployment_types.go

```diff
@@ -258,7 +258,7 @@ func TestDeploymentExpectedReplicas(t *testing.T, ctx *TestingContext) {
 				continue
 			}
 
-			if deployment.Status.Replicas != product.ExpectedReplicas {
+			if deployment.Status.Replicas != product.ExpectedReplicas && deployment.Status.Replicas < product.ExpectedReplicas {
```

Comment:
Just wondering is the first condition,`deployment.Status.Replicas != product.ExpectedReplicas`, of the `if` blocks still necessary ? 

I think `deployment.Status.Replicas < product.ExpectedReplicas` would cover the check for replicas to be greater or equal to the expected number of replicas :thinking:

Reply:
@MStokluska

Reply:
Thanks @KevFan that makes sense! updated the PR